### PR TITLE
Add the release doc for the Kvrocks controller

### PR DESCRIPTION
**Pull request by git-hulk**
image

### CODE REVIEW

**Comment by git-hulk** [comment]
Those lines are copied from the Kvrocks release except changing the release repo to Kvrocks controller.

File: community/create-a-kvrocks-controller-release.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+```html
+[ANNOUNCE] Release Apache Kvrocks Controller ${release_version}
+
+Hi all,
+
+The Apache Kvrocks community is pleased to announce
+that Apache Kvrocks Controller ${release_version} has been released!
+
+Kvrocks is a distributed key value NoSQL database that uses RocksDB
```
Comment (PragmaTwice) [inline community/create-a-kvrocks-controller-release.md:187]: Introduction for kvrocks controller?

Comment (git-hulk) [reply community/create-a-kvrocks-controller-release.md:187]: Sure, will change to Kvrocks controller introduction and add the download page.

**Comment by git-hulk** [comment]
@PragmaTwice @torwig I haven't added the controller download page to the sidebar yet since we don't have any release for now. Will add it after releasing our first formal release.

**Review by PragmaTwice — APPROVED** [review]

**Pull request merged**